roles/lib_openshift/src/generate.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
+'''
+  Generate the openshift-ansible/roles/lib_openshift_cli/library/ modules.
+'''
+
+import argparse
+import os
+import StringIO
```

Comment:
This is causing a python3-related test failure in travis. Instead of this, you can use:
```
from six import StringIO
```
and change later `StringIO.StringIO` references to just `StringIO`.